torch/onnx/symbolic_caffe2.py

```diff
@@ -154,6 +154,24 @@ def upsample_nearest2d(g, input, output_size, align_corners=None, scales_h=None,
     output = nhwc2nchw(g, output)
     sym_help._quantized_ops.add(output)
     return output
+@parse_args('v', 'is', 'is', 'is', 'is', 'i')
+def max_pool2d(g, input, kernel_size, stride, padding, dilation, ceil_mode):
+    if input not in sym_help._quantized_ops:
+        from torch.onnx.symbolic_opset9 import max_pool2d
+        return max_pool2d(g, input, kernel_size, stride, padding, dilation, ceil_mode)
+    kwargs = {
+        "strides_i": stride,
+        "pads_i": padding + padding,
+        "kernel_i": kernel_size[0],
+        "order_s": "NHWC",
+        "Y_scale_f": input.node()["Y_scale"],
+        "Y_zero_point_i": input.node()["Y_zero_point"],
+    }
+    input = nchw2nhwc(g, input)
```

Comment:
Is there a way to check if it is already in NHWC? That way we could add a fast pass here

Reply:
The way this conversion flow is written, it always adds these permutes before and after conv ops. Else branches in the graph become hard to handle. Once we convert the model we can use a script to remove the unnecessary permutes from the graph.